README.md

```diff
@@ -395,6 +393,8 @@ This project welcomes contributions and suggestions.
 **Here are some 
 [code standards](docs/developer/code_standard.rst) when you submit a pull request.**
 
+If you want to modify the qlib document, you can follow the following figure.
```

Comment:
You don't have to post such a big image.
You can use a smaller figure.

For example, such size is more reasonable.  
![image](https://user-images.githubusercontent.com/465606/134899467-8bf6b78c-484d-44f0-9090-671570635091.png)